Sources/XCTest/XCTestSuite.swift

```diff
@@ -10,28 +10,15 @@
 //  XCTestSuite.swift
 //  A collection of test cases.
 //
+// A concrete subclass of XCTest, XCTestSuite is a collection of test cases.
+// Suites are usually managed by the IDE, but XCTestSuite can also be
+// instantiated and manipulated directly:
+//
+//     let suite = XCTestSuite(name: "My Tests")
+//     suite.addTest(myTest)
+//     suite.testCaseCount // 1
+//     suite.perform(myRunner)
```

Comment:
I'd write this as `suite.run()` instead. Alternatively, this could be `suite.perform(myTestRun)`.